common/containers/Tabs/Contracts/index.jsx

```diff
@@ -0,0 +1,126 @@
+// @flow
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  accessContract,
+  deployContract,
+  fetchNodeContracts
+} from 'actions/contracts';
+import { State } from 'reducers/contracts';
+import translate from 'translations';
+import Interact from './components/Interact';
+import Deploy from './components/Deploy';
+import './index.scss';
+
+type Props = {
+  nodeContracts: Array,
+  selectedAddress: ?string,
+  selectedABIJson: ?string,
+  selectedABIFunctions: ?Array,
+  accessContract: Function,
+  deployContract: Function
+};
+
+class Contracts extends Component {
+  props: Props;
+  static propTypes = {
+    // Store state
+    nodeContracts: PropTypes.array.isRequired,
+    selectedAddress: PropTypes.string,
+    selectedABIJson: PropTypes.string,
+    selectedABIFunctions: PropTypes.array,
+    // Actions
+    fetchNodeContracts: PropTypes.func.isRequired,
+    accessContract: PropTypes.func.isRequired,
+    deployContract: PropTypes.func.isRequired
+  };
+
+  state = {
+    activeTab: 'interact'
```

Comment:
Yeah, this makes a lot of sense, to have each have its own route (But still use the same component.) I'll look into this.